docs/source/conf.py

```diff
@@ -51,7 +56,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns: List[str] = []
```

Comment:
Not sure if our docs (or sphinx for that matter) also benefit from typing... but I don't mind it!

Reply:
It's tricky to make mypy ignore top level definitions like this in strict mode, so it's easier to fix it than ignore it

Reply:
I usually call mypy just on `src/` to keep it a bit narrower - usually examples, docs, tests are not that critical for typing a user-facing API.